packages/starlight/utils/navigation.ts

```diff
@@ -308,6 +321,8 @@ function applyPrevNextLinkConfig(
 				...link,
 				label: config.label ?? link.label,
 				href: config.link ?? link.href,
+				// Explicitly remove sidebar link attributes for prev/next links.
+				attrs: undefined,
```

Comment:
Ah, although I guess this would diverge here then. Maybe resetting to `attrs: {}` could make sense? What are your thoughts?

Reply:
Makes sense to me 👍